lib/routes/withdraw/reverse_swap/confirmation_page/reverse_swap_confirmation_page.dart

```diff
@@ -90,7 +87,10 @@ class _ReverseSwapConfirmationPageState extends State<ReverseSwapConfirmationPag
 
   void _fetchReverseSwapFeeOptions() {
     final reverseSwapBloc = context.read<ReverseSwapBloc>();
-    _fetchFeeOptionsFuture = reverseSwapBloc.fetchReverseSwapFeeOptions(sendAmountSat: widget.amountSat);
+    _fetchFeeOptionsFuture = reverseSwapBloc.fetchReverseSwapFeeOptions(
+      amountSat: widget.amountSat,
+      amountType: widget.isMaxValue ? SwapAmountType.Receive : SwapAmountType.Send,
```

Comment:
Shouldn't this be the other way around?

```dart
amountType: widget.isMaxValue ? SwapAmountType.Send : SwapAmountType.Receive
```

Reply:
🤔 I don't think so.
If we're sending maximum value, we'd like to drain our balance or send maximum amount within policy. 
Let's say our balance is 58k, we're interested in getting fees for senderAmountSat:58k.

Reply:
If the user wants to drain all channel (e.g 58k) then the user wants to say "I want to send 58k how much do I receive at the end after fees)"
So the for max amount the type should be `Send`.

Reply:
You're both correct. I made my comment looking at the snippet Carlos attached as the correct example, not the implementation 😅 

Fixed c9cf3750b1b76f6065c3b1535992c6ee20b903d9